src/macros.jl

```diff
@@ -950,6 +950,61 @@ function namecall(basename, idxvars)
     return ex
 end
 
+reverse_sense(::Val{:<=})   = :>=
+reverse_sense(::Val{:≤})    = :≥
+reverse_sense(::Val{:>=})   = :<=
+reverse_sense(::Val{:≥})    = :≤
+reverse_sense(::Val{:(==)}) = :(==)
+
+"""
+    parse_one_operator_variable(_error::Function, infoexpr::VariableInfoExpr, sense::Val{S}, value) where S
+
+Update `infoexr` for a variable expression in the `@variable` macro of the form `variable name S value`.
+"""
+parse_one_operator_variable(_error::Function, infoexpr::VariableInfoExpr, ::Union{Val{:<=}, Val{:≤}}, upper) = setupperbound_or_error(_error, infoexpr, upper)
+parse_one_operator_variable(_error::Function, infoexpr::VariableInfoExpr, ::Union{Val{:>=}, Val{:≥}}, lower) = setlowerbound_or_error(_error, infoexpr, lower)
+parse_one_operator_variable(_error::Function, infoexpr::VariableInfoExpr, ::Val{:(==)}, value) = fix_or_error(_error, infoexpr, value)
+parse_one_operator_variable(_error::Function, infoexpr::VariableInfoExpr, ::Val{S}, value) where S = _error("Unknown sense $S.")
+function parsevariable(_error::Function, infoexpr::VariableInfoExpr, sense::Symbol, var, value)
+    # Variable declaration of the form: var sense value
+
+    # There is not way to determine at parsing time which of lhs or rhs is the
```

Comment:
It might be confusing that this works but then it does not work when it is not a constant.
We might take advantage of the case where it is a constant to warn the user that this is not correct.